scripts/us_census/acs5yr/subject_tables/common/README.md

```diff
@@ -25,3 +25,230 @@ python3 common/process.py --output=[OUTPUT] --download_id=[DOWNLOAD_ID] --featur
 ```
 
 This will generate the files `output.tmcf` and `output.csv` in the path specified by the `output` flag.
+
+
+## Spec Validator
+
+### Description
+
+The validator tool tries to do basic sanity checks on the spec.
+
+The script parses:
+- The set of CSV files downloaded from census (Any one)
+	- List of CSV file paths
+	- List of ZIP file paths
+	- JSON file containing list of all columns
+- Config Spec JSON file
+
+It tries to identify:
+- Tokens appearing in spec but not in any of the CSV files
+- Tokens appearing in CSV files but not in spec
+- Columns that have no property assignment
+- Columns that might possibly be in conflict with ignore and property assignment
+- Missing Denominator Total Columns
+- Denominator ccolumn appearing under multiple totals
+- Possible missing EnumSpecialisations
+- Multiple measurement assignment
+- Multiple population assignment
+- Extra inferredSpec
+
+And creates:
+- List and count of all column names
+- List and count of all ignored column names
+- List and count of all accepted column names
+
+The script generates column and token data file and another file with outcome of results in the output directory
+
+`all` key of dictionary refers to combination across all the files in the input
+
+filename based outcomes can also be found in the output files
+
+`filewise` option is generally used for debug purposes only
+
+`check_metadata` for zip file, `is_metadata` for CSV files can be made True if _metadata_ type files need to be processed
```

Comment:
Are these command line options or should they be added to a config file?

Reply:
filewise option can be accessed when calling the function externally only, while is_metadata can be used with function call as well as command line. Updated the README to reflect this.